endpoints/openrtb2/auction.go

```diff
@@ -905,6 +911,12 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 	if hasStoredBidRequest {
 		resolvedRequest, err = jsonpatch.MergePatch(storedRequests[storedBidRequestId], requestJson)
 		if err != nil {
+			if !json.Valid(requestJson) {
+				err = errBadIncomingRequest
+			} else if !json.Valid(storedRequests[storedBidRequestId]) {
+				err = errBadStoredRequest
```

Comment:
It may be better to have this dynamic, and report back the stored request ID that is causing an issue. More of an issue on the bad stored Imp than the bad stored request, as a request could be referencing multiple stored Imps